packages/api/amplify_api/lib/src/api_plugin_impl.dart

```diff
@@ -140,6 +150,20 @@ class AmplifyAPIDart extends AmplifyAPI {
     }
   }
 
+  void _emitHubEvent(WebSocketState state) {
```

Comment:
At some point we should add `apiName` to the events so they can be distinguished when multiple blocs are running. Can you add a TODO for now?